packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -300,26 +517,56 @@ class ViewSettingsDialog extends UI5Element {
 	 * Stores current settings as confirmed and fires <code>confirm</code> event.
 	 */
 	_confirmSettings() {
-		this._confirmedSettings = this._currentSettings;
-		this.fireEvent("confirm", {
-			sortOrder: this._confirmedSettings.sortOrder && this._confirmedSettings.sortOrder.innerText,
-			sortBy: this._confirmedSettings.sortBy ? this._confirmedSettings.sortBy.innerText : "",
-		});
 		this.close();
+		this._confirmedSettings = this._currentSettings;
```

Comment:
_confirmedSettings sounds like (sorting + filtering), but we have _confirmedFilters in addition,
And same foes for  _restoreSettings and _restoreFilters

We should think of better naming to improve the readability of the code.

Reply:
Done